syntax = "proto3";

package worker;

// Importa le definizioni di altri file .proto se necessario
// import "google/protobuf/any.proto";


message Worker {
  string id = 1;
  string host = 2;
  int32 port = 3;
  WorkerStatus status = 4;
}

enum WorkerStatus {
  IDLE = 0;
  IN_PROGRESS = 1;
  COMPLETED = 2;
  FAILED = 3;
}

